.circleci/config.yml

```diff
@@ -297,6 +298,15 @@ commands:
           var_file: ./variables.tfvars
 
 jobs:
+  secrets-check:
+    executor: machine-executor
+    branches:
+    steps:
+      - checkout
+      - docker-compose-check
```

Comment:
Did this fail without the `docker-compose-check`? I don't see docker being used in the script so I think its unneeded here.

Reply:
No, I don't think it's necessary now that you point it out. I'll remove it.